core/rfb.js

```diff
@@ -863,6 +864,10 @@ export default class RFB extends EventTargetMixin {
                 this._rfb_auth_scheme = 16; // Tight
             } else if (includes(2, types)) {
                 this._rfb_auth_scheme = 2; // VNC Auth
+            } else if (includes(17, types)) {
+                this._rfb_auth_scheme = 17; // UltraVNC
+            } else if (includes(104, types)) {
+                this._rfb_auth_scheme = 104; // UltraVNC SC
```

Comment:
Do you know of some documentation for these extensions? Do you think you could write up a patch for @rfbproto?

Reply:
Ok, I will try to write it there.